### ISSUE TITLE
can't run thecode MOFF score -i MOFFscore_test.txt

### ISSUE THREAD

**Issue by FlorenceDecoy** [MDhewei/MOFF#3]
ValueError: Kernel shape must have the same length as input, but received kernel of shape (1, 19, 3) and input of shape (None, None, 16, 19).when i run the code MOFF score -i MOFFscore_test.txt,i encountered this error,Is the input data provided incorrect?

**Comment by AlonLuboshitz** [comment]
Yes happens to me to. i was able to run the model before. maybe something with tesnorflow, keras updates. Anyway they should check it because it happend when lloading the g3.h5 model
